runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropConfig.java

```diff
@@ -75,4 +75,27 @@ default Color getMagePrayerColor()
 	{
 		return new Color(0x15, 0x80, 0xAD);
 	}
+
+	@ConfigItem(
+			keyName = "showFakeXpDrops",
```

Comment:
Fix these indents, see our [code style wiki](https://github.com/runelite/runelite/wiki/Code-Conventions) page and import the XML linked there.